From 24c2ed85973ed7dd17787cf49631d06f3549bd83 Mon Sep 17 00:00:00 2001
From: Wasim Nazir <quic_wasimn@quicinc.com>
Date: Mon, 11 Nov 2024 17:15:05 +0530
Subject: [PATCH 3/3] PENDING: arm64: dts: qcom: qcs9075-rb8: Enable llcc node

Add last level cache controller for qcs9075-rb8.
This enables llcc[0,1,2,3,4,5]_base & llcc_broadcast_base.

Signed-off-by: Wasim Nazir <quic_wasimn@quicinc.com>
Upstream-Status: Pending
---
 arch/arm64/boot/dts/qcom/qcs9075-rb8.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/qcom/qcs9075-rb8.dts b/arch/arm64/boot/dts/qcom/qcs9075-rb8.dts
index b27966cfdf7b..4798c09e529f 100644
--- a/arch/arm64/boot/dts/qcom/qcs9075-rb8.dts
+++ b/arch/arm64/boot/dts/qcom/qcs9075-rb8.dts
@@ -352,6 +352,10 @@ &i2c19 {
 	qcom,xfer-mode = <1>;
 };
 
+&llcc {
+	status = "okay";
+};
+
 &pcie0 {
 	perst-gpios = <&tlmm 2 GPIO_ACTIVE_LOW>;
 	wake-gpios = <&tlmm 0 GPIO_ACTIVE_HIGH>;
-- 
2.48.1

